docs/simulation/workflow.md

```diff
@@ -66,8 +67,9 @@ Like in a 2-color FRET experiment, horizontal dimensions of the video are equall
 Single molecules are then spread randomly on the donor channel and directly translated into the acceptor channel.
 
 At molecule coordinates, pixel values are set to donor or acceptor pure fluorescence intensities, including donor anisotropy and setup cross-talks.
-Channel-specific background and uniform camera noise are added to all pixels to convert fluorescence to camera-detected signal. 
-Finally, pixels are convolved with channel-specific point spread functions to obtain realistic diffraction-limited images. 
+Channel-specific fluorescent background is added to consider all sources of detected lights. 
```

Comment:
But it has to be photons right? Because it's how it is in the simulation and should be mentioned somehow

Reply:
Its photons in any way, otherwise, it is related to the camera noise model (like a bg due to read-out noise which is of course different from "real" photon background)

Background: stray light, scattered light, (auto-)fluorescence etc.